src/main/java/net/mcreator/element/types/Plant.java

```diff
@@ -202,6 +202,10 @@ public boolean doesGenerateInWorld() {
 		return spawnWorldTypes.size() > 0;
 	}
 
+	public String getRenderType() {

```

Comment:
Maybe add this method to the IBlock interface so all blocks are inherently forced to have this property?